### DISCUSSION TITLE
CMS Validator tool and provider groups with the same TIN

### DISCUSSION THREAD

**Discussion by jltang-kp** [CMSgov/price-transparency-guide-validator#58] [Q&A]
We are running into an error code calling out duplicates using the validator tool. Specifically with provider groups that share the same TIN but different NPIs listed in the NPI array.

Just to confirm this is an error on the validator tool side right? @shaselton-usds  The one of the reasons why we would have multiple provider objects with the same TIN but different NPI arrays are to separate specialist and PCP rates.

![image](https://user-images.githubusercontent.com/85644623/174665268-0965705e-2d73-4ab7-8f20-d4c3c9623b6f.png)

**Comment by shaselton-usds** [comment]
@jltang-kp Thanks for the feedback. Would it be possible to share the code snippet of "duplicate" TINs with different NPIs?

Going to move this discussion over to the validator repo too...
